doc/whats_new.rst

```diff
@@ -212,6 +212,8 @@ BUG
 
     - Fix treatment of vector inverse in :func:`mne.minimum_norm.apply_inverse_epochs` by `Emily Stephen`_
 
+    - Fix parallel processing when computing covariance with shrinkage estimators by `Denis Engemann`_
```

Comment:
is this still true with this reduced PR?

Reply:
Yes, it’s for the last cov PR we merged.